lib/multiplex-transformers.cc

```diff
@@ -235,5 +235,63 @@ Canvas *ZStripeTransformer::Transform(Canvas *output) {
   return canvas_;
 }
 
+// ------------------------
+// Coreman Panel 1:8 32x32
+
+class CoremanTransformer::TransformCanvas : public BasicMultiplexCanvas {
+public:
+  TransformCanvas(int panel_rows, int panel_cols)
+    : BasicMultiplexCanvas(panel_rows, panel_cols) {}
+
+  virtual void SetPixel(int x, int y, uint8_t r, uint8_t g, uint8_t b) {
+    if (x < 0 || x >= width_ || y < 0 || y >= height_) return;
+
+    const int chained_panel  = x / panel_cols_;
+    const int parallel_panel = y / panel_rows_;
+
+    const int within_panel_x = x % panel_cols_;
+    const int within_panel_y = y % panel_rows_;
+
+    const bool is_top_check = (within_panel_y % (panel_rows_/2)) < panel_rows_/4;
+    const bool is_left_check = (within_panel_x < panel_cols_/2);
+    int new_x, new_y;
+    if (is_top_check) {
+      new_x = is_left_check ? within_panel_x+panel_cols_/2 : within_panel_x+panel_cols_;
+    } else {
+      new_x = is_left_check ? within_panel_x : within_panel_x + panel_cols_/2;
+    }
+    new_y = (within_panel_y / (panel_rows_/2)) * (panel_rows_/4)
+      + within_panel_y % (panel_rows_/4);
+
+   if ((y <= 7) || ((y >= 16) && (y <= 23)) || ((y >= 32) && (y <= 39)) || ((y >= 48) && (y <= 55)) || ((y >= 64) && (y <= 71)) || ((y >= 80) && (y <= 87))) {
```

Comment:
Given that you only can do maximum 32 in one panel, this if can be formulated much simpler by just using `within_panel_y` instead of just `y` (and it would also make it working if parallel chains can ever be more than 3).

Also, try to formulate your expression not with constants (such as 7, 16, 23...), but derived from panel_rows_ and panel_cols_. That way, this transformer will also work with a 32x16 panel that has the similar mapping, not only 32x32 panels.